### ISSUE TITLE
Communicate “page breaks” via more than just styles

### ISSUE THREAD

**Issue by chasenlehara** [CCALI/a2jdeps#51]
As a screen reader user, I want page breaks to be available in text, so that I can perceive them.

Currently, page breaks in the preview document are only perceivable through visual styles. They should also be available in the text a screen reader announces to users.

---

At a minimum, I think we should add some sort of aria element to indicate page breaks to screen readers.

The draft WCAG 2.2 spec adds [Success Criterion 2.4.13 Page Break Navigation](https://www.w3.org/TR/WCAG22/#page-break-navigation) and includes these docs on [understanding the criterion](https://www.w3.org/WAI/WCAG22/Understanding/page-break-navigation.html) and [techniques for meeting the criterion](https://w3c.github.io/wcag/techniques/html/H99).

I think the minimum I mentioned above is sufficient for compliance, although we may consider adding the recommended navigation to the document. My only hesitation is that the navigation would not necessarily match what’s in the generated PDF document—I think we would need to use more advanced techniques to determine the pages that will be in the generated PDF.

**Comment by chasenlehara** [comment]
@mikemitchel and I talked about this—it should be sufficient for us to just announce page breaks to screen readers and not add nav to “pages.”

**Comment by chasenlehara** [comment]
It looks like my initial fix did not work as expected.

1. The `sr-only` class didn’t seem to take effect (maybe `bootstrap-styles` needs to be added somewhere?).

2. There are other page breaks created by CKEditor that aren’t fixed; the DOM looks like this:

```html
<div style="page-break-after:always"><span style="display:none"> </span></div>
```
